src/OrchardCore/OrchardCore.Abstractions/Shell/ShellSettings.cs

```diff
@@ -33,6 +33,7 @@ public ShellSettings(ShellSettings settings)
         {
             _settings = new ShellConfiguration(settings._settings);
             _configuration = new ShellConfiguration(settings.Name, settings._configuration);
+            TenantId = settings.TenantId;
```

Comment:
Anyway we don't need this line as it comes from the `_settings` `ShellConfiguration` that we init just before by using the `ShellSettings settings` parameter from which we build a new `ShellSettings`.